docs/shrc_diag_util.md

```diff
@@ -262,106 +282,170 @@ Line number information for classes in the cache is then shown with one of the f
+- `Zip cache bytes`
+    - The number of zip entry cache bytes stored in the cache.
+- `Startup hint bytes`
+    - The number of bytes of data stored to describe startup hints.
+- `Data bytes`
+    - The number of bytes of non-class data stored by the VM.
+- `Metadata bytes`
+    - The number of bytes of data stored to describe the cached classes. :fontawesome-solid-pencil-alt:{: .note aria-hidden="true"} **Note:** This field is available only in the top layer cache output or when a cache is not layered.
+- `Metadata % used`
+    - The proportion of metadata bytes to class bytes, which indicates how efficiently cache space is being used. The value shown does consider the `Class debug area size`.
+- `Class debug area size`
+    - The size in bytes of the Class Debug Area. This area is reserved to store `LineNumberTable` and `LocalVariableTable` class attribute information.
+- `Class debug area bytes used`
+    - The size in bytes of the Class Debug Area that contains data.
+- `Class debug area % used`
+    - The percentage of the Class Debug Area that contains data.
+- `ROMClasses`
+    - The number of classes in the cache. The cache stores `ROMClasses` \(the class data itself, which is read-only\) and information about the location from which the classes were loaded. This information is stored in different ways, depending on the Java `SharedClassHelper` API that was used to store the classes. For more information, see [Support for custom class loaders](shrc.md#support-for-custom-class-loaders).
+- `AOT methods`
+    - Optionally, `ROMClass` methods can be compiled and the AOT code stored in the cache. The `AOT methods` information shows the total number of methods in the cache that have AOT code compiled for them. This number includes AOT code for stale classes.
+- `Classpaths`, `URLs`, and `Tokens`
+    - The number of class paths, URLs, and tokens in the cache. Classes stored from a `SharedClassURLClasspathHelper` are stored with a Classpath. Classes stored using a `SharedClassURLHelper` are stored with a URL. Classes stored using a `SharedClassTokenHelper` are stored with a Token. Most class loaders, including the bootstrap and application class loaders, use a `SharedClassURLClasspathHelper`. The result is that it is most common to see class paths in the cache.
+    - The number of Classpaths, URLs, and Tokens stored is determined by a number of factors. For example, every time an element of a Classpath is updated, such as when a `.jar` file is rebuilt, a new Classpath is added to the cache. Additionally, if partitions or modification contexts are used, they are associated with the Classpath, URL, or Token. A Classpath, URL, or Token is stored for each unique combination of partition and modification context. For more information, see [Sharing modified bytecode and `SharedClassHelper` cache partitions](shrc.md#sharing-modified-bytecode).
+- `Zip caches`
+    - The number of `.zip` files that have entry caches stored in the shared cache.
```

Comment:
This is expected to be zero for Java 11 and above, unless a jar file is added to the bootclasspath.